src/App.js

```diff
@@ -4,20 +4,22 @@ import MockMan from "mockman-js";
 import {Routes,Route} from "react-router";
 import Home from "./pages/Home/Home";
 import { ProductPage } from "./pages/ProductPage/ProductPage";
-import { SingleProductDetail } from "./pages/singleProductDetail/singleproductDetail";
+import { LogInPage } from "./pages/LogIN/LogInPage";
```

Comment:
as we go further there may be many component Pages to import so make it online using an ```index.js``` file instead

Reply:
done